src/native/windows/apps/prunsrv/prunsrv.c

```diff
@@ -1838,8 +1859,9 @@ void WINAPI serviceMain(DWORD argc, LPTSTR *argv)
         SetConsoleCtrlHandler((PHANDLER_ROUTINE)console_handler, TRUE);
 
         if (SO_STOPTIMEOUT) {
-            /* we have a stop timeout */
+            /* we wait for ever for the service to be done, printing a debug message every 2 seconds */
```

Comment:
Doesn't this wait for `timeout` seconds rather than waiting forever?

Reply:
We need to wait for ever until a stop service is issued and there we wait for the timeout...

Reply:
Maybe update the comment to reflect that? "for the service to be done" suggests "until the service has stopped" rather than "until the call to stop the service has returned".